paddle/framework/net.h

```diff
@@ -101,5 +101,7 @@ class PlainNet : public Net {
   }
 };
 
+std::shared_ptr<PlainNet> AddBackwardOp(std::shared_ptr<PlainNet> ForwardOps);
```

Comment:
I think this method might be useless, we can directly invoke `OpRegistry::CreateGradOp(...)`